hibernate-reactive-core/src/main/java/org/hibernate/reactive/pool/impl/SqlClientConnection.java

```diff
@@ -264,4 +273,58 @@ public boolean hasNext() {
 	public CompletionStage<Void> executeBatch() {
 		return voidFuture();
 	}
+
+	private static void translateNulls(Object[] paramValues) {
```

Comment:
If I understand correctly, this method will be invoked regardless of the DB vendor? In this case, it is a problem because `NullValue` so far is only used in MS SQL Client.

Reply:
Well, I mean, we tested its and nothing broke...

Reply:
Yeah, not breaking `Tuple` behavior was a goal of the PR that introduced `NullValule` :-)

Sorry for the confusion